### ISSUE TITLE
[Bug] [a11y]Menu Settings-Multiple item labels affected, considered not readable by screen readers

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#9678]
## Prerequisites
- Have Google Accessibility Scanner installed.
## Steps to reproduce
1. Open app and tap on the 3 dot menu.
2. Scan the app.
### Expected behavior
There are no suggestions to make any changes.
### Actual behavior
The following settings have the following suggestion: This item may not have a label readable by screen readers.
- Item label - Library
[548,730][1404,894]
- Item label - Add-Ons
[548,894][1404,1058]
- Item label - Quit
[548,1222][1404,1386]
- Item label - Report Site Issue
[548,1389][1404,1553]
- Item label - Find in page
[548,1553][1404,1717]
- Item label - Add to top sites
[548,1717][1404,1881]
- Item label - Save to collection
[548,2045][1404,2209]
### Device information

* Android device: Samsung Galaxy S9 (Android 8)
* Fenix version: Firefox Preview Nightly 4/3 (Build #20940609)

Screen:
![screenshot_Firefox Nightly_2020-04-03-12_21_07](https://user-images.githubusercontent.com/49859863/78379340-e3007600-75da-11ea-9c74-472c17cf72bf.png)

**Comment by asadotzler** [comment]
I've just tested our main menu with a build from 4/3 and all of the labels are read by Talkback and Voice Assistant. Unless I'm missing something, this is a false positive from the testing software.

**Comment by asadotzler** [comment]
Unable to reproduce. Closing issue.

**Comment by sflorean** [comment]
I am able to reproduce this on Samsung Galaxy Note 10 (Android 10), latest Nightly 5/13, and Beta build 76.0.0-beta.1 only with **Google Accessibility Scanner** installed. 
Verified with Voice Assistant ON, and the items were readable.
